### PR TITLE
update: 'Other Deck Actions'

### PR DESCRIPTION
**Pull request by snowtimeglass**
This commit will update the section of long-press-deck menu.

### CODE REVIEW

File: manual.asc [hunk 1]
```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
...
 Please see the https://docs.ankiweb.net/deck-options.html#deck-options[desktop documentation] for more information about these study options.
 
 Custom study :: Allows you to choose from some convenient presets for studying outside of your normal schedule, for example increasing the study limit for the day. See the section on <<filtered,"filtered decks">> for more detailed information.
 
-Delete deck :: Use this option to delete a deck (note: this action is not reversible, although you can <<backups,restore from a backup>>)
-
 Export deck :: This option can be used to share a deck with other users. See the <<exporting, exporting decks>> section for more information.
 
+Create shortcut :: Allows you to add a shortcut icon of the selected deck to the home screen of the system (Android)
+
```
Comment (david-allison) [inline manual.asc:99]:
```suggestion
Create shortcut :: Adds a shortcut to study the deck on the Android home screen

```

File: manual.asc [hunk 2]
```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
...
+Edit description :: Use this option to create or edit additional text that is shown on the <<deckOverview, deck overview>> screen of the selected deck
```
Comment (david-allison) [inline manual.asc:100]:
```suggestion
Edit description :: Create or edit text shown on the deck's <<deckOverview, deck overview>> screen
```

File: manual.asc [hunk 3]
```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
...
+
+Delete deck :: Use this option to delete a deck (note: this action is not reversible, although you can <<backups,restore from a backup>>)
```
Comment (david-allison) [inline manual.asc:102]:
```suggestion
Delete deck :: Delete a deck (note: this action is not reversible, although you can <<backups,restore from a backup>>)
```

Comment (david-allison) [reply manual.asc:102]: Is this true? Anki's undo should work.

File: manual.asc [hunk 4]
```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
 === Other Deck Actions
 Long tapping on a deck will show a list of other actions available to perform on that deck:
 
-Rename deck :: Use this option to rename a deck
+Add :: Use this option to create a new empty note to the seleted deck
```
Comment (david-allison) [inline manual.asc:83]:
```suggestion
Add :: Add a new note to the selected deck
```

File: manual.asc [hunk 5]
```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
...
+
+Browse cards :: Use this option to see the cards of the selected deck with the <<browser, card browser>>
```
Comment (david-allison) [inline manual.asc:85]:
```suggestion
Browse cards :: View all cards of the deck in the <<browser, card browser>>
```

File: manual.asc [hunk 6]
```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
...
+
+Rename deck :: Use this option to rename the selected deck
```
Comment (david-allison) [inline manual.asc:87]:
```suggestion
Rename deck :: Renames the selected deck
```

Comment (david-allison) [reply manual.asc:87]: You may want to link to a section on deck naming here. This is how people add subdecks

**Review by david-allison — CHANGES_REQUESTED** [review]
Few suggestions, mostly making things more terse (`this option` adds no value to me as a qualifier, it's obvious what the context of the sentence is)

File: manual.asc (hunk 3 above)
Comment (snowtimeglass) [reply manual.asc:102]: This isn't true now. It is a part of text in the current manual, and it should be fixed. 

Though I didn't write this part (I just changed the order of the item), I overlooked it. Thank you for pointing it out.

Comment (snowtimeglass) [reply manual.asc:102]: How about this change:
`Delete deck :: Deletes the selected deck. (You can undo this action by 'Undo' button in the popup message shown after the action, or 'Undo Delete Deck' in the screen menu.)`

File: manual.asc (hunk 6 above)
Comment (snowtimeglass) [reply manual.asc:87]: How about this change:
```
Rename deck :: Renames the selected deck

NOTE: You can create a subdeck in the selected deck, 
and also can make the selected deck a subdeck of another deck, 
by using the following naming convention: "PARENT::CHILD" (i.e. "French::Idiom").
For more information, please see 
https://docs.ankiweb.net/getting-started.html#decks[the section in Anki manual].
```

Comment (david-allison) [reply manual.asc:87]: That looks great, thanks @snowtimeglass

File: manual.asc (hunk 3 above)
Comment (david-allison) [reply manual.asc:102]: That's great!

**Review by david-allison — APPROVED** [review]
Thanks! Needs a squash merge

**Comment by snowtimeglass** [comment]
Thank you, I'll do it.

(I wasn't confident that the rebase is appropriate in this case as well, as this is my first experience with "committing suggestions by reviewers" procedure.)

**Review by mikehardy — APPROVED** [review]
Thanks for all the manual cleanup! Much appreciated

**Pull request merged**